lib/node-utils.js

```diff
@@ -588,15 +588,16 @@ function getTokenType(token) {
  * @returns {ESTreeToken}       the converted ESTreeToken
  */
 function convertToken(token, ast) {
-    const start = token.getStart(),
+    const start = (token.kind === SyntaxKind.JsxText) ? token.getFullStart() : token.getStart(),
+        end = token.getEnd(),
         value = ast.text.slice(start, token.end),
```

Comment:
All the other references to `token.end` were updated to the variable `end`, but not this one. Just checking that is intended/not significant?